test/unit_test/rag/test_naive_merge_docx.py

```
@@ -0,0 +1,185 @@
+sized close to chunk_token_num, a chunk can grow to nearly double the budget
+before the next iteration notices.
+"""
+
+import re
+
+import pytest
+
+from rag import nlp
+from rag.nlp import _merge_cks, naive_merge_docx
+
+DEFAULT_DELIMITER = "\n。；！？"
```

Ruff (0.16.1)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)